roles/ceph-dashboard/tasks/configure_dashboard.yml

```diff
@@ -211,12 +193,31 @@
       with_items: "{{ groups[iscsi_gw_group_name] }}"
       run_once: true
 
-- name: inject grafana dashboard layouts
-  command: "{{ container_exec_cmd }} ceph --cluster {{ cluster }} dashboard grafana dashboards update"
-  delegate_to: "{{ groups[mon_group_name][0] }}"
-  run_once: true
-  changed_when: false
-  when: containerized_deployment | bool
+- name: set dashboard endpoints and inject layouts
+  block:
+  - name: set grafana url
```

Comment:
these vars will just be left set to the value generated on the last node which executed the block though; it'd be interesting to know which other command (in addition to "dashboards update") will consume them

Reply:
Exactly, because if the purpose of these commands are merely related to the dashboard injection, we should unset these endpoints after pushing them (see my previous comment). 
Each grafana instance points to the local prometheus and alertmanager (we see this from the config files) and all of them have the same view of the cluster and collect data provided by the node-exporter: what I'm trying to say is that I don't see any data coming from a sampling process (passing from the mgr dashboard to the grafana instance).